.evergreen/print-compass-env.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-export MONGODB_DEFAULT_VERSION=6.0.x
+export MONGODB_DEFAULT_VERSION=7.0.x
```

Comment:
I'm thinking we should probably not set this here and use it print-compass-env.js and then somehow rather fill in a default if nothing is set in the compass tests. That would be the only way to have a consistent default locally and in CI because we don't use this file when just doing `npm test` locally.